cmd/osm-controller/osm-controller.go

```diff
@@ -128,6 +136,7 @@ func main() {
 	}
 	kubeClient := kubernetes.NewForConfigOrDie(kubeConfig)
 	policyClient := policyClientset.NewForConfigOrDie(kubeConfig)
+	apiServerClient := apiServerClientset.NewForConfigOrDie(kubeConfig)
```

Comment:
Since the reconciler for CRDs doesn't happen inside osm-controller, can you pass this as nil and let the reconciler not reconcile the CRD if set to nil?

Reply:
thanks for the suggestion, I've updated the code